apps/public/src/app/(general)/contact/Contact.tsx

```diff
@@ -4,15 +4,22 @@ import { Alert, Heading, Label, Paragraph, TextInput } from '@amsterdam/design-s
 import { Grid, SubmitButton } from '@meldingen/ui'
 import { useActionState } from 'react'
 
+import type { StaticFormTextAreaComponentOutput } from 'apps/public/src/apiClientProxy'
+
 import { BackLink } from '../_components/BackLink'
 
 import { postContactForm } from './actions'
 
 const initialState: { message?: string } = {}
 
-export const Contact = () => {
+export const Contact = ({ formData }: { formData: StaticFormTextAreaComponentOutput[] }) => {
   const [formState, formAction] = useActionState(postContactForm, initialState)
 
+  const emailLabel = formData[0].label
+  const emailDescription = formData[0].description
+  const telLabel = formData[1].label
+  const telDescription = formData[1].description
+
   return (
     <Grid paddingBottom="large" paddingTop="medium">
       <Grid.Cell span={{ narrow: 4, medium: 6, wide: 6 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
```

Comment:
This component needs internationalisation

Reply:
Yup. I didn't do that yet to prevent merge conflicts.